### ISSUE TITLE
Editor : Button zur Textbereinigung

### ISSUE THREAD

**Issue by stephan-harmanus** [KplusH/jsbach-timeline#61]
Wordformattierungen werden übernommen und können nicht gelöscht werden. Bitte Button zum Entfernen von unliebsamen Fomrattierungen im Editor einfügen

**Comment by Andreas-Schoenefeldt** [comment]
<img width="569" alt="screen shot 2017-11-06 at 13 49 20" src="https://user-images.githubusercontent.com/1157580/32441791-548ea96e-c2f9-11e7-92ad-c5ffa68dec5f.png">

Die Schriftgröße vom Zitat Block sollte ebenfalls angepasst werden.

**Comment by stephan-harmanus** [comment]
Ja, wenn man diese Wordformatierungen "times" rausnimmt und die Hausschrift erscheint, also di "Klavika", dann ist der Zitatblock größer.

**Comment by stephan-harmanus** [comment]
<img width="646" alt="bildschirmfoto 2017-11-06 um 17 09 12" src="https://user-images.githubusercontent.com/23217462/32450736-47f1b50e-c315-11e7-8ade-39ae11a7144e.png">

Hab die Wordformatierung gelöscht. Jetzt siehst Du hier den vergrößerten Zitatblock

**Comment by Andreas-Schoenefeldt** [comment]
Hi Stephan, 
also das css vom blockquote ist jetzt an dden normalen text angepasst :)

Zur Formatbereinigung gibt es leider ohne Weiteres nur den manuellen Button:

<img width="363" alt="screen shot 2017-11-07 at 12 29 34" src="https://user-images.githubusercontent.com/1157580/32491913-b3353626-c3b8-11e7-93c1-94348d7fc58c.png">

Man kann das theoretisch auch hacken, würde da aber lieber auf eine saubere Lösung warten.

Liebe Grüße
Andreas

**Comment by Andreas-Schoenefeldt** [comment]
A way to include beautufull " marks? maybe as Sonderzeichen?
